lib/radioactive_bunnies/worker.rb

```diff
@@ -1,17 +1,14 @@
 require 'atomic'
 require 'radioactive_bunnies/context'
-
+require 'radioactive_bunnies/deadletter_worker'
 module RadioactiveBunnies::Worker
 
-  def ack!
-    true
-  end
-
   def work
   end
 
   def self.included(base)
     base.extend ClassMethods
+    base.extend RadioactiveBunnies::DeadletterWorker::ClassMethods
```

Comment:
Disregard. Made sense when I read the spec example below. We might want to consider a way to opt out of bloating the object unnecessarily, but its negligible at the moment.